core/src/components/divider/divider.tsx

```diff
@@ -1,7 +1,7 @@
 import { Component, Prop, Host, h } from '@stencil/core';
 
 @Component({
-  tag: 'sdds-divider',
+  tag: 'tds-divider',
   styleUrl: 'divider-component.scss',
```

Comment:
I think you can rename the scss file to divider.scss to conform with our code standards.